ee/ucp/user-access/kubectl.md

```diff
@@ -83,7 +83,7 @@ You can download the binary from this [link](https://storage.googleapis.com/kube
 If you have curl installed on your system, you use these commands in Powershell.
 
 ```cmd
-$env:k8sversion = "v1.8.11"
+$env:k8sversion = "v1.11.5"
```

Comment:
@david-yu the current copy of the page had the kubectl version set to `1.11.5`. So we just applied the same change for our Windows code block.

Reply:
Sounds good